README.md

```diff
@@ -19,7 +19,8 @@ npm run build
 
 ```
 npm install
-npm run start:web
+npm run build
```

Comment:
Thanks! It should be `build:web` though to make it work

Reply:
Thanks, updated.